### DISCUSSION TITLE
AudiobookConverter is damaged and can't be opened

### DISCUSSION THREAD

**Discussion by sreza4** [yermak/AudioBookConverter#376] [Q&A]
Hello,
I keep getting this error. I am running MacOS Monterey 12.1. Anyone have a solution?

**Comment by patg75** [comment]
In the installation wiki, there's how to start it via terminal. 
https://github.com/yermak/AudioBookConverter/wiki/Installation#macos-x

Make sure, that in SystemPreferences > Security & Privacy > Developer Tools > Terminal is checked.

That said, I get it to run, but can't add any files. Too many exception errors.

**Reply by patg75** [reply]
PS. macOS never asked me to set permissions for the app, and I have no clue which ones would be needed...

**Comment by mrkellar** [comment]
Got it to run in Sonoma using the above suggestion but sadly can't add any files.

**Reply by likebliss** [reply]
I also got it to run, but can't add any files.  Don't know where to approve permissions, not sure if that's the cause?
